mappymatch/utils/plot.py

```diff
@@ -178,3 +192,91 @@ def plot_map(tmap: NxMap, m=None):
         ).add_to(m)
 
     return m
+
+
+def plot_match_distances(coord_df):
+    # todo MatchResult is not a defined Element, removed from the args list above (was (matches: MatchResult)).
+    # build and display plots here.
+    """
+    Summary: Plot the points deviance from known roads with matplotlib.
+
+    Issues:
+        we have two different dataframes one labeled with gdf and the other with df. --> to resolve this, the gdf labeled data frames have been changed to df labels.
+
+    Args:
+        coord_df (pandas dataframe): coords of guessed points in the area.
+    """
+
+    # Road Data Frame Section
+    # Define a pandas data frame containing the list of matches (each represented by m) where m.road = True
+
+    # mid_i = int(
+    #     len(coord_df) / 2
+    # )  # find the middle index of the coord_gdf data frame.
+    # mid_coord = coord_df.iloc[
+    #     mid_i
+    # ].geometry  # answer the question: what is the middle coordinate?
+
+    y = coord_df.distance  # the distances from the expected line. Deviance.
```

Comment:
Here, we could just pull the distances directly from the matches, something like:

```python
y = [m.distance for m in matches]
```